spec/truffle/launcher_spec.rb

```diff
@@ -72,4 +72,16 @@ def should_print_full_java_command(cmd)
     end
   end
 
+  it "takes options from system properties" do
+    out = `TRUFFLERUBYOPT=-J-Dtruffleruby.verbosity=1 #{RbConfig.ruby} -e 'puts $VERBOSE'`
+    $?.success?.should == true
+    out.should == "false\n"
```

Comment:
`$VERBOSE` as `false` is the default, so this does not test properly. Maybe use `=2`/-W2/`true` for the test?